charts/signoz/values.yaml

```diff
@@ -1853,6 +1860,14 @@ k8s-infra:
       # -- Service Type: LoadBalancer (allows external access) or NodePort (more secure, no extra cost)
       type: ClusterIP
 
+    # -- OtelAgent daemonset annotation.
+    annotations: {}
+    # -- OtelAgent pod(s) annotation.
+    podAnnotations:
+      signoz.io/scrape: 'true'
+      signoz.io/port: '8888'
+      signoz.io/path: /metrics
```

Comment:
I am not following this. Otel agent pod annotation should be part of k8s-infra, right?

Reply:
yes, they are. K8s-infra chart default values are overridden in signoz values.yaml.
I just included it here so that users can opt out of the metrics collection from signoz chart values.yaml as well.